docs/pages/docs/guides/custom-admin-ui-pages.mdx

```diff
@@ -14,38 +16,163 @@ Much like with Next.js, all files in this directory will be added as routes to t
+            <h1>This is a custom Admin UI Page</h1>
+            <p>It can be accessed via the route `/custom-page`</p>
+        </PageContainer>
+    )
+}
+```
+
+Much better, our custom page looks and feels like an Admin UI page now.
+![custom page with correctly styled header](/assets/guides/custom-admin-ui-pages/custom-page-with-styled-header.png)
+
+## Custom route in Admin UI Navigation
+
+Yes, our custom page is looking pretty great, and much more like an Admin UI page, but it's not visible as a navigation item.
+We can fix this by adding a custom Navigation component with a route pointing to our custom page.
+
+First add the following files to the `/admin` directory in the root of your Keystone project.
+
+```tsx
+// admin/config.ts
+import type { AdminConfig } from '@keystone-next/types';
+import { CustomNavigation } from './components/CustomNavigation';
+export const components: AdminConfig['components']= {
+    Navigation: CustomNavigation
+};
+```
+
+```tsx
+// admin/components/CustomNavigation.tsx
+import { NavigationContainer, ListNavItems, NavItem } from '@keystone-next/keystone/admin-ui/components';
+import type { NavigationProps } from '@keystone-next/keystone/admin-ui/components';
+export function CustomNavigation({ lists, authenticatedItem }: NavigationProps) {
+    return (
+        <NavigationContainer authenticatedItem={authenticatedItem}>
+            <NavItem href="/">Dashboard</NavItem>
+            <ListNavItems lists={lists} />
+        </NavigationContainer>
+    )
+}
+```
+
+!> We've skipped over the specifics for this guide, if you're interested in more detail on creating a custom Navigation component check out the [Custom Admin UI Navigation](/docs/guides/custom-admin-ui-navigation) guide.
+
+Lastly we'll add our new route to the newly created `CustomNavigation` component.
+
+```tsx
+// admin/components/CustomNavigation.tsx
+import { NavigationContainer, ListNavItems, NavItem } from '@keystone-next/keystone/admin-ui/components';
+import type { NavigationProps } from '@keystone-next/keystone/admin-ui/components';
+export function CustomNavigation({ lists, authenticatedItem }: NavigationProps) {
+    return (
+        <NavigationContainer authenticatedItem={authenticatedItem}>
+            <NavItem href="/">Dashboard</NavItem>
+            <ListNavItems lists={lists} />
+            <NavItem href="/custom-page">Custom Page</NavItem>
+        </NavigationContainer>
+    )
+}
+```
+
+!> Under the hood Keystone's Admin UI is powered by Next.js, so the route to our custom page is the filename of our custom page component, in this case its `/custom-page`
+
+With all that in place, our custom Admin UI page is now navigable from the Admin UI Navigation component, and we can access it from other pages in the Admin UI.
+![completed custom Admin UI page](/assets/guides/custom-admin-ui-pages/custom-page-completed.png)
+
+!> This concludes the main part of the Custom Admin UI Pages guide
+
```

Comment:
```suggestion
```

I think we can let people bounce from the document whenever they need without explicitly calling out checkpoints.